lib/fizzy/capi.cpp

```diff
@@ -576,17 +578,17 @@ FizzyInstance* fizzy_resolve_instantiate(const FizzyModule* c_module,
     }
 }
 
-void fizzy_free_instance(FizzyInstance* instance)
+void fizzy_free_instance(FizzyInstance* instance) noexcept
```

Comment:
Are we saving anything with marking all these noexcept?

If I understand correctly:
- before this, if used in C, an exception just causes UB (abort effectively), while in C++ it is passed through
- after this, if used in C and C++, an exception is a clear abort (however we generate exception handling code in each of these functions?)